app/helpers/users_helper.rb

```diff
@@ -4,4 +4,8 @@ module UsersHelper
   def current_user_name(user)
     [user.name, user.email].find(&:present?)
   end
+
+  def user_name_or_email(user)
+    user.name.nil? || user.name == '' ? user.email : user.name
```

Comment:
```suggestion
    user.name.presence || user.email
```

と書くことができます。

https://qiita.com/jnchito/items/dedb3b889ab226933ccf#%E7%A9%BA%E3%81%AA%E3%82%89%E5%88%A5%E3%81%AE%E5%80%A4%E3%82%92%E4%BB%A3%E5%85%A5%E3%81%AE%E4%BB%A3%E3%82%8F%E3%82%8A%E3%81%ABpresence%E3%82%92%E4%BD%BF%E3%81%86